shared/src/ledger/queries_ext.rs

```diff
@@ -165,6 +168,9 @@ pub trait QueriesExt {
         &'db self,
         epoch: Option<Epoch>,
     ) -> Box<dyn Iterator<Item = (EthAddrBook, Address, token::Amount)> + 'db>;
+
+    /// Retrieve the `max_proposal_bytes` consensus parameter from storage.
+    fn get_max_proposal_bytes(&self) -> ProposalBytes;
```

Comment:
for sure `QueriesExt` can be broken up/refactored/done away with soon! The API of the `Storage` struct grows further...

Reply:
lol yeah

Reply:
I'm actually deleting it. It will become `PosQueries`. Any funcitonality that isn't that will have to live elsewhere.